features/270-next-skycli/commands.md

```diff
@@ -0,0 +1,517 @@
+
+Change skycli current app context to `APP_NAME`
+
+### skycli config set-cluster-server
+
+`skycli config set-cluster-server [CLUSTER_SERVER_URL]`
+
+Change current cluster controller server endpoint, it is provided by cluster
+admin.
+
+## skycli user
+
+### skycli user create
+
+`skycli user create`
+
+Cluster admin only. Create cluster user, need to provide email and password interactively.
+
+### skycli user list
+
+`skycli user list`
+
+Cluster admin only. List all cluster user.
+
+## skycli app
+
+### skycli app list
+
+`skycli app list`
+
+List all user's app. If using admin key to call this app, list all the apps in
+the cluster instead.
+
+### skycli create app
+
+`skycli create app`
+
+Create app, the skycli current app context will be updated to the newly created
+app.
+
+### skycli app add-user
+
+`skycli app add-user [USER_EMAIL]`
+
+Add user into the current app. Change current app by
+`skycli config set-app`.
+
+### skycli app view-tenant-config
+
+`skycli app view-tenant-config`
+
+View the current app tenant config in yaml format.
+
+#### Example
+
+```
+$ skycli app view-tenant-config
+- MASTER_KEY: xxxxx
+- API_KEY: xxxxx
+- SSO:
+  ALLOWED_CALLBACK_URLS:
+  - http://127.0.0.1:5000/
+  - http://localhost:5000/
+  MERGE_USERS_WITH_IDENTICAL_EMAIL: true
+```
+
+### skycli app update-tenant-config
+
+`skycli app update-tenant-config -f [TENANT_SETTING_YAML_FILE]`
+
+Update tenant config interactively or by providing a file. For the config format,
+please see [user facing config](user-facing-config.md)
+
+#### Example
+
+**Update in editor**
+```sh
+$ skycli app update-tenant-config
+? Edit application tenant config. Press <enter> to launch your preferred editor.
+
+# Enter editor mode with existing config
+- MASTER_KEY: xxxxx
+- API_KEY: xxxxx
+- SSO:
+  ALLOWED_CALLBACK_URLS:
+  - http://127.0.0.1:5000/
+  - http://localhost:5000/
+  MERGE_USERS_WITH_IDENTICAL_EMAIL: true
+  ...
+~
+~
+~
+~
+~
+~
+~
+```
+
+**Update by providing a file**
+```sh
+# 1. If you don't have a copy of the tenant config file, get it from the view command.
+$ skycli app view-tenant-config > ./tenant-config.yaml
+
+# 2. Update the tenant-config.yaml file
+
+# 3. Apply the change to the app
+$ skycli app update-tenant-config -f ./tenant-config.yaml
+```
+
+### skycli domain
+
+#### Overview
+
+### skycli domain add
+
+`skycli domain add [CUSTOM_DOMAIN]`
+
+This command will only add domain into the app. The domain is not usable in
+this stage. User need to verify the domain.
+
+#### Example
+
+```
+$ skycli domain add api.example.com
+Added domain api.example.com successfully!
+
+1. Add the TXT record below to your DNS provider to verify your own example.com
+
+    Type    Host            Value
+    TXT     example.com     skygear-cloud-verification=5636486ffc5a4dfebf4a13f480bd9a95
+
+2. Add `CNAME` record to make sure your domain is pointing to `cf.skygearapis.io`.
+
+After updating DNS records, run `now domain verify api.example.com` to verify domain.
```

Comment:
`now domain verify api.example.com` should be `skicli verify api.example.com`.

Reply:
Updated, thanks @kaojohnny